mlops_proj/src/evaluate/evaluate.py

```
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def evaluate(model, val_loader):
+    total_loss = 0
+    all_predictions = []
+
+    for features, labels in val_loader:
```

Ruff (0.12.2)

Loop control variable `labels` not used within loop body

(B007)